src/content/configuration/output.md

```diff
@@ -1055,7 +1056,7 @@ T> It also adds some info about tree shaking to the generated bundle.
 
 ## `output.publicPath`
 
-`string = ''` `function`
+`string = 'auto'` `function`
```

Comment:
```suggestion
`string` `function`
```
It depends. As per https://github.com/webpack/webpack/releases/tag/v5.0.0-beta.32:

> `output.publicPath` is now `"auto"` by default when supported by target

And here's the code setting default value for it https://github.com/webpack/webpack/blob/master/lib/config/defaults.js#L648, which means it defaults to `auto` in some cases while `''` in other cases.